### PR TITLE
Change rating models to have one boolean field

### PR DESCRIPTION
**Pull request by hnlee**
Change Rating to have one boolean field, `helpful`, instead of `kind`, `specific`, `actionable`
Remove some duplicated tests

### CODE REVIEW

File: app/views/reviews/show.html.erb [hunk 1]
```diff
@@ -2,9 +2,7 @@
 <br />
 
 <% @ratings.each do |rating| %>
-  Kind: <%= rating.kind %><br />
-  Specific: <%= rating.specific %><br />
-  Actionable: <%= rating.actionable %><br />
+  Kind, specific, actionable: <%= rating.helpful %><br />
```
Comment (NicoleCarpenter) [inline app/views/reviews/show.html.erb]: I am assuming this will go away once the 👍 👎  thing happens?

Comment (hnlee) [inline app/views/reviews/show.html.erb]: Yup, just a placeholder for now.

File: spec/integration/project_index_spec.rb [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-  it 'shows all projects as links' do
-    project1 = create(:project)
-    project2 = create(:project, title: "Java Tic-Tac-Toe")
-
-    visit "/"
-
-    page.has_content? project1.title
-    page.has_content? project2.title
-  end
-end
```
Comment (NicoleCarpenter) [inline spec/integration/project_index_spec.rb]: Why is this being deleted?

File: spec/integration/project_show_spec.rb [hunk 3]
```diff
@@ -1,39 +0,0 @@
...
-  end
-
-  describe 'if not from a project page' do
-    it 'redirects to project index' do
-      visit new_review_path
-
-      expect(current_path).to eq(projects_path)
-    end
-  end
-end
```
Comment (NicoleCarpenter) [inline spec/integration/project_show_spec.rb]: Why is this being deleted?

File: spec/integration/project_index_spec.rb (hunk 2 above)
Comment (hnlee) [inline spec/integration/project_index_spec.rb]: Duplicated test that exists in spec/integration/project_spec.rb

File: spec/integration/project_show_spec.rb (hunk 3 above)
Comment (hnlee) [inline spec/integration/project_show_spec.rb]: Duplicated test that exists in spec/integration/project_spec.rb

File: spec/integration/project_index_spec.rb (hunk 2 above)
Comment (NicoleCarpenter) [inline spec/integration/project_index_spec.rb]: 👍

File: app/views/reviews/show.html.erb (hunk 1 above)
Comment (NicoleCarpenter) [inline app/views/reviews/show.html.erb]: 👍

**Comment by NicoleCarpenter** [comment]
👍

**Pull request merged**